### ISSUE TITLE
Specify next steps with Liquid Democracy specification

### ISSUE THREAD

**Issue by medied** [DemocracyEarth/contracts#3]
Luke (Aragon) shared with us the initial [Liquid Democracy specification](https://docs.google.com/document/d/1fw_gpxaMsV-tJHpKz8jU22cciLBREQ5vB3T6nvvJbtU/edit?ts=5a832ca7). Currently the doc is being discussed and reviewed, we waiting on Luke to specify next steps. 

@virgile-dev @herbstephens @raulk feel free to leave comments about the doc in this this thread or within the doc itself.

**Comment by medied** [comment]
Update: meeting had been postponed and Github repo not created yet

<img width="700" alt="screen shot 2018-03-21 at 9 42 38 am" src="https://user-images.githubusercontent.com/4752599/37713168-6c7bc5ea-2cec-11e8-8266-85cb3a7458e4.png">
